qttypes/src/qtcore/qstring.rs

```diff
@@ -317,6 +317,12 @@ impl TryFrom<QString> for i16 {
     }
 }
 
+impl From<QString> for Vec<u8> {
+    fn from(x: QString) -> Self {
+        QByteArray::from(x).into()
```

Comment:
First you convert from QString to QByteArray,  and then again to to Vec, that's two allocation.
You'd be better of using Rust's DecodeUtf16
(or passing through String instead)